app/src/main/java/com/randomappsinc/studentpicker/choosing/ChoicesDisplayViewHolder.kt

```diff
@@ -0,0 +1,30 @@
+package com.randomappsinc.studentpicker.choosing
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import butterknife.BindView
+import butterknife.ButterKnife
+import com.randomappsinc.studentpicker.R
+import com.randomappsinc.studentpicker.models.ListInfo
+
+class ChoicesDisplayViewHolder(
+        view: View,
+        private val listInfo: ListInfo
+) {
+
+    @BindView(R.id.names_list) lateinit var namesList: RecyclerView
+
+    var chooseNameDialogAdapter: ChooseNameDialogAdapter
+
+    init {
+        ButterKnife.bind(this, view)
```

Comment:
Can we not use Android View Binding instead of ButterKnife? Not a huge deal, just wondering.